.github/workflows/downstream-project-spl-nightly.yml

```diff
@@ -6,13 +6,7 @@ on:
 
 jobs:
   main:
-    # As this is a cron job, it is better to avoid running it for all the forks.
-    # They are unlike to benefit from these executions, and they could easily
-    # eat up all the minutes GitHub allocation to free accounts.
-    if: >
-      github.event_name != 'schedule'
```

Comment:
don't we want `github.event_name != 'schedule'` anymore?

Reply:
afaik the `schedule` is the only event that calls this pipeline atm. I think it should be okay to include it here. I can split this into another PR if that would make things clearer!